management/project/migration/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Migration between regions
+permalink: /management/project/migration/
+---
+
+* TOC
+{:toc}
+
+Keboola Connection is available in multiple [regions](https://developers.keboola.com/overview/api/#regions-and-endpoints) — currently in the US (connection.keboola.com) and EU (connection.eu-central-1.keboola.com). Each region instance is a completely independent full stack of Keboola Connection services.
+Project is always located in one of KBC stacks (regions). In case you need to migrate project for whatever reason between regions you can achieve that in cooperation with [Keboola Support](/management/support/).
+
+## Migration steps
+
+You can proceed a migration in following steps:
+
+- Obtain a new empty project in destination region.
+- Contact Keboola support from source project and provide a destination project.

```

Comment:
- Obtain a new empty project in the destination region
- Contact Keboola support using the `support` link in your source project and provide the name and ID of your destination project